### DISCUSSION TITLE
Energy-Optimal Trajectory Planning for Multirotor Drones (Project 237)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#73] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 237](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/blob/main/projects/Energy-Optimal%20Trajectory%20Planning%20for%20Multirotor%20Drones/README.md)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by imthiasa** [comment]
Hi, I am working in RL, I want to explore the potential of MATLAB
